Source/PlayField.cpp

```diff
@@ -0,0 +1,102 @@
+//
+// Created by Sam Petrov on 22.04.2020.
+//
+#include <vector>
+#include <cassert>
+#include "../Headers/PlayField.h"
+
+enum class PlayField::csState{
+        csEmpty,
+        csCross,
+        csNought
+};
+enum class PlayField::fnState{
+        fsInvalid,
+        fsNormal,
+        fsCrossesWin,
+        fsNoughtsWin,
+        fsDraw
+};
+
+PlayField::PlayField() {
+    previosState = csState::csCross;
+    for(int i = 0; i < 9; ++i)
+        cellField[i] = csState::csEmpty;
+}
+
+PlayField::CellPos::CellPos(int a, int b) {
+    assert((a >= 0 && a <= 2) && (b >= 0 && b <= 2));
```

Comment:
```suggestion
    assert( (0<=a && a<someSize) && (0<=b && a<someSize) );
```
А то вырвиглазно. Почему-то все пишут как вы...